grpc/python/BUILD

```diff
@@ -38,13 +39,13 @@ python_grpc_compile(
 python_repackage(
     name = "protocol_pkg",
     src = ":protocol_src",
-    package = "grakn_protocol"
+    package = "graknprotocol"
 )
 
 py_library(
     name = "protocol",
     srcs = ["protocol_pkg"],
-    imports = ["../../grpc/python"]
+    imports = ["../../../grpc/python"]
```

Comment:
This should always have been the import path- without this change, the pip package was putting `graknprotocol` inside `bazel-out/darwin-fastbuild/...` which was causing the IDE to fail to resolve the import.